rules/ekscluster-kfd-v1alpha2.yaml

```
@@ -52,3 +49,22 @@ distribution:
     reducers:
       - key: distributionModulesTracingTempoBackend
         lifecycle: pre-apply
+  - path: .spec.distribution.modules.dr.type
+    immutable: false
+    description: "changes to the dr module type have been detected. This will cause the replacement of the current DR (velero) stack (removal or creation)."
+    reducers:
+      - key: distributionModulesDRType
+        lifecycle: pre-apply
+  - path: .spec.distribution.modules.ingress.nginx.type
+    immutable: false
+    description: "changes to the nginx type in the ingress module have been detected. This will cause the reconfiguration or deletion of the current policy ingress stack."
+    unsupported:
+      - to: single
+        from: dual
+        reason: "currently, switching from dual to single is not supported. You need to first remove the current stack with type: none."
+      - to: dual
+        from: single
+        reason: "currently, switching from single to dual is not supported. You need to first remove the current stack with type: none."
+    reducers:
+      - key: distributionModulesIngressNginxType
+        lifecycle: pre-apply
```

nitpick: missing new line
```suggestion
        lifecycle: pre-apply

```